src/node/desktop/package-lock.json

```diff
@@ -3227,6 +3227,30 @@
         "readable-stream": "^3.4.0"
       }
     },
+    "node_modules/bl/node_modules/buffer": {
```

Comment:
I would expect no changes to this file for this PR, given nothing has been touched under src/node/desktop?

Reply:
It looks like this matches the current state on main, though, probably reflecting the same changes from here:

https://github.com/rstudio/rstudio/commit/adab76b727f64150841a634150684e01c8f492d5

I think I had run into `npm ci` complaining that `package-lock.json` was out-of-sync, and so had to update things to get into sync again?